src/ui-lib/sass/06-utilities/_is-hidden.scss

```diff
@@ -0,0 +1,3 @@
+.is-hidden {
```

Comment:
Is this being used? If so I think it needs to be name `.grav-u-is-hidden` and perhaps have `!important` since it will be a trump.

Reply:
not actually used, I think I'm going to remove it for now 👍